frontend/components/containers/Fellow/GithubProfile.jsx

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 
-export default function GithubProfile({ info }) {
-  console.log(info);
+const GithubProfile = ({ accountInfo }) => {
+  const { Name, Location, Bio } = accountInfo;
 
   return (
     <StyledLink href="https://github.com/sebastiancrossa" target="_blank">
```

Comment:
Hey @sebastiancrossa we need to update this profile links to users. I didn't catch it earlier, can you update this in a follow up PR?